[ONNX] Fix split export in opset13 (#56277)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#57605 [ONNX] Fix split export in opset13 (#56277)**

Fix split export in opset13

Co-authored-by: Ksenija Stanojevic <ksenija.stanojevic@gmail.com>
Co-authored-by: BowenBao <bowbao@microsoft.com>

Differential Revision: [D28393522](https://our.internmc.facebook.com/intern/diff/D28393522)

KsenijaS committed:
[ONNX] Fix split export in opset13 (#56277)

Fix split export in opset13

Co-authored-by: Ksenija Stanojevic <ksenija.stanojevic@gmail.com>
Co-authored-by: BowenBao <bowbao@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Fix split export in opset13 (#56277)"


Fix split export in opset13

Co-authored-by: Ksenija Stanojevic <ksenija.stanojevic@gmail.com>
Co-authored-by: BowenBao <bowbao@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Fix split export in opset13 (#56277)"


Fix split export in opset13

Co-authored-by: Ksenija Stanojevic <ksenija.stanojevic@gmail.com>
Co-authored-by: BowenBao <bowbao@microsoft.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Fix split export in opset13 (#56277)"


Fix split export in opset13

Co-authored-by: Ksenija Stanojevic <ksenija.stanojevic@gmail.com>
Co-authored-by: BowenBao <bowbao@microsoft.com>

[ghstack-poisoned]

SplitInfinity commented:
@SplitInfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28393522).

Commit:
Update on "[ONNX] Fix split export in opset13 (#56277)"


Fix split export in opset13

Co-authored-by: Ksenija Stanojevic <ksenija.stanojevic@gmail.com>
Co-authored-by: BowenBao <bowbao@microsoft.com>

Differential Revision: [D28393522](https://our.internmc.facebook.com/intern/diff/D28393522)

[ghstack-poisoned]
